OADB/AliPhysicsSelection.cxx

```diff
@@ -157,24 +167,22 @@ fCashedTokens(NULL)
  fPSOADB(0),
  fFillOADB(0),
  fTriggerOADB(0),
- fRegexp(new TPRegexp("([[:alpha:]]\\w*)")),
- fCashedTokens(NULL)
+ fTriggerToFormula()
  {
    // constructor
    fCollTrigClasses.SetOwner(1);
    fBGTrigClasses.SetOwner(1);
    fTriggerAnalysis.SetOwner(1);
    fHistList.SetOwner(1);
-
+   fTriggerToFormula = new StringToTFormula();
    AliLog::SetClassDebugLevel("AliPhysicsSelection", AliLog::kWarning);
  }
 
 AliPhysicsSelection::~AliPhysicsSelection(){
   if (fPSOADB)       delete fPSOADB;
```

Comment:
Since you already modifying code around here, I think this might be a great opportunity to remove these unnecessary null-pointer checks.